twisted/topfiles/8760.feature

```diff
@@ -0,0 +1 @@
+Client and server TLS connections now support ChaCha20 ciphers when available from OpenSSL.
```

Comment:
I think describining it more abstractly is good, but perhaps be more
specific: Default configuration for client and server TLS connections [...].

On Mon, Aug 15, 2016 at 9:44 AM, Cory Benfield <notifications@github.com>
wrote:

> In twisted/topfiles/8760.feature
> <https://github.com/twisted/twisted/pull/499#discussion_r74762855>:
>
> > @@ -0,0 +1 @@
> > +Client and server TLS connections now support ChaCha20 ciphers when available from OpenSSL.
>
> I 100% get that. My main concern is that most users are hopefully not
> using CertificateOptions directly but are instead using endpoints that
> configure it for them, and so are likely more interested in the affect on
> those endpoints.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/twisted/twisted/pull/499/files/64d2ef29ee0651d142c35a506fe87e9d1e3c3106#r74762855>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAADBBVMbev4ARkiZxXi2FxYKF6DuvJzks5qgG1agaJpZM4JkX-j>
> .
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6